logic.js

```diff
@@ -19,7 +19,7 @@
 const sha256 = require("bcrypto").sha256;
```

Comment:
@edisonljh I think we should use `hashjs` for consistency.

Reply:
Working on it now, will push to the next PR.